department-of-veterans-affairs/vets-api
# Add job to initialize uvac statsd metrics

## Summary
Adds Sidekiq scheduled job that runs on startup. The job will initialize uvac statsd keys to their current value. Job will be disabled/removed after first run

- Added scheduled job that will run on startup
- This job will be disabled after first run
- Code will be removed after run/disable

## How to test
- Start rails server - `bundle exec rails`
- Start sidekiq - `bundle exec sidekiq`
- Check statsD logs - you will see keys like `user_avc_updater.*.*.added`
- Check development.log - you will see log with message `UserAcceptableVerifiedCredential - StatsD Initialized`
- Navigate to http://localhost:3000/sidekiq/recurring-jobs
- Find `UserAcceptableVerifiedCredentialInitStatsd` and disable it
- Restart rails server and sidekiq
- Check logs - no new StatsD nor development log message should have been created

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#0000
- *Link to ticket created in va.gov-team repo*
- *Link to previous change of the code/bug (if applicable)*
- *Link to epic if not included in ticket*


## Testing done

- Added specs
- Tested locally

## Screenshots
_Note: Optional_


## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria


## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
